client/reader/search-stream/index.jsx

```diff
@@ -126,7 +126,7 @@ class SearchStream extends React.Component {
 
 		let searchPlaceholderText = this.props.searchPlaceholderText;
 		if ( ! searchPlaceholderText ) {
-			searchPlaceholderText = translate( 'Search billions of WordPress.com posts…' );
+			searchPlaceholderText = translate( 'Search billions of WordPress posts…' );
```

Comment:
:disappointed: 25 existing translations will be lost with this change.

Reply:
Sorry @a8ci18n!